eng/scripts/npm/pack-workspace.mjs

```diff
@@ -0,0 +1,88 @@
+import { resolve, dirname } from 'path';
+import { execSync } from 'child_process';
+import fsExtra from 'fs-extra';
+const { existsSync, writeJsonSync, readJsonSync, moveSync } = fsExtra;
```

Comment:
Nitpick, and feel free to ignore, but isn't this equivalent to:

```suggestion
import { existsSync, writeJsonSync, readJsonSync, moveSync } from 'fs-extra';
```

Reply:
Nope in node world. I had to explicitly change this. I don't know the exact reasons, but it has to do with the fact that it's a  commonjs module being imported in an es6 context (which node 20+ supports)